### DISCUSSION TITLE
GLSP Theia client: Define and populate custom pane from GLSP Java Server?

### DISCUSSION THREAD

**Discussion by mrcalvin** [eclipse-glsp/glsp#1306] [General]
As an extension to https://github.com/eclipse-glsp/glsp/discussions/1305: How would I add a custom pane (next to Problems and Output) and send data to it?

**Comment by planger** [comment]
To create a custom view in Theia, please follow the [Theia documentation on that topic](https://theia-ide.org/docs/widgets/).
To send data from GLSP to this Theia widget or vice versa, actually the same approach applies as in https://github.com/eclipse-glsp/glsp/discussions/1305#discussioncomment-9152245.
